php/Terminus/Commands/SitesCommand.php

```diff
@@ -522,17 +523,18 @@ private function getSiteCreateOptions($assoc_args) {
       $message .= " (if left blank will be $suggested_name)";
 
       $options['name'] = $this->input()->string(
-        array(
-          'args'    => $assoc_args,
-          'key'     => 'site',
-          'message' => $message,
-          'deafult' => $suggested_name,
-        )
+        [
+          'args'     => $assoc_args,
+          'key'      => 'site',
+          'deafult'  => $suggested_name,
+          'message'  => $message,
+          'required' => true,
```

Comment:
Does this interfere with the message above of `(if left blank will be $suggested_name)"`